openshift/template.yaml

```diff
@@ -101,6 +101,24 @@ objects:
                     name: assisted-installer-rds
               - name: IMAGE_BUILDER_CMD
                 value: ""
+- apiVersion: v1
+  kind: Service
+  metadata:
+    labels:
+      app: bm-inventory
+    name: bm-inventory
+    namespace: assisted-installer
+  spec:
+    ports:
+      - name: bm-inventory
+        port: 8090
+        protocol: TCP
+        targetPort: 8090
+    selector:
+      app: bm-inventory
+    type: LoadBalancer
```

Comment:
it should be cluster ip

Reply:
The type is taken from `bm-inventory:deploy/bm-inventory-service.yaml` and their it is `LoadBalancer`.
1. do you still want me to change it to `ClusterIp`?
2. do we need to make the same change in `bm-inventory`'s repo?
3. why ClusterIp and not LoadBalancer?

Reply:
Load Balancer means that it will provisioning a public IP (which we don't want).

We need to change it to cluster ip as default and not load balancer. please raise it in the daily meeting